R/count_labels.R

```diff
@@ -36,6 +36,7 @@
 #'   countsnow = unname(counts)
 #' )
 #'
+#' library(shiny)
```

Comment:
In this example packages are attached as they are needed. In others all packages are attached at the beginning of examples. 
I understand that both ways can be appropriate. Would you mind making sure that it makes sense in all cases?

Reply:
They are all attached as needed, not necessary before the first call though.

The reasoning was to either:
- place it before the set of expressions that use it
- or before the ShinyApp is built (this case)

![image](https://github.com/insightsengineering/teal.slice/assets/211358/47f2d220-bc90-44f7-b338-38157929ab79)

This example is ambiguous though, I'll move it to the top just in case

Reply:
Good enough for me :+1: